### ISSUE TITLE
MOD SUPPORT SUGGESTIONS

### ISSUE THREAD

**Issue by MehVahdJukaar** [MehVahdJukaar/WoodGood#77]
Add here some mods you would like to be added. Note that not all might be added immediately or at all depending on complexity, popularity of the request and iof they fit my selection criteria. These include having some blocks that can be easily recolored and can work without needing custom textures as the mod of course can only recolor things. Also things like entities could be added but I dont have a system in place yet so only blocks will be considered for now

**Comment by murao-kun** [comment]
Done! **Woodworks**
*Curseforge*: https://www.curseforge.com/minecraft/mc-mods/woodworks
*Blocks*: Bookshelves, Chests, Ladders, Beehives, Boards and Leaf piles.
*Versions*: 1.18.2 / 1.19.2
*Loaders*: Forge

**Comment by murao-kun** [comment]
Done! **Infinity Buttons**
*Curseforge*: https://www.curseforge.com/minecraft/mc-mods/infinity-buttons
*Blocks*: Large wooden buttons.
*Versions*: 1.18.2 / 1.19.2
*Loaders*: Forge / Fabric

**Comment by WenXin20** [comment]
[**Functional Storage**](https://www.curseforge.com/minecraft/mc-mods/functional-storage)
_Blocks_: Drawers
_Versions_: 1.18 / 1.19
_Loaders_: Forge

**Comment by murao-kun** [comment]
**Mosaic Carpentry**
*Curseforge*: https://www.curseforge.com/minecraft/mc-mods/mosaic-carpentry
*Blocks*: Mosaic block, mosaic stairs, mosaic slab.
*Versions*: 1.18.2 / 1.19.2
*Loaders*: Forge

**Comment by murao-kun** [comment]
**Boatload**
*Curseforge*: https://www.curseforge.com/minecraft/mc-mods/boatload
*Blocks*: boat with furnace, large boat, large boat with chest, large boat with furnace.
*Versions*: 1.18.2 / 1.19.2 (Recommend to only do 1.19.2, since it's change a lot compare to 1.18.2)
*Loaders*: Forge

**Comment by murao-kun** [comment]
**Every-compat** itself
*Curseforge*: https://www.curseforge.com/minecraft/mc-mods/every-compat
*Blocks*: Crafting table. (Useful for building)
*Versions*: 1.18.2 / 1.19.2
*Loaders*: Forge / Fabric

**Comment by Hypsellis** [comment]
**Quark**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/quark
_Blocks:_ leaf carpet (they are currently missing)
_Versions:_ 1.18.2 / 1.19.2
_Loaders:_ Forge

**Comment by 0Navis0** [comment]
**Macaw's Roofs**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/macaws-roofs
_Blocks:_ roof, attic roof, cube roof, top roof, lower roof, steep roof
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by 0Navis0** [comment]
**Macaw's Furnitures**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/macaws-furniture
_Blocks:_ table, desk, drawer, wardrobe, cabinet, counter, chair
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by 0Navis0** [comment]
**Quark**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/quark
_Blocks:_ leaf carpet
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by Hypsellis** [comment]
**WindSwept**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/windsweptmod
_Blocks:_ holly wood, chestnut wood
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by MehVahdJukaar** [comment]
Added quark leaf carped and wood works stuff (copie pasted from quark module so yay double blocks). Chests not included as they are harder to do and the ones I have now entirely rely on quark code

**Comment by emoace** [comment]
**Carpenter**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/carpenter
_Blocks:_ chests, bookshelves
_Versions:_ 1.18.2
_Loaders:_ Forge/Fabric

**Comment by enduum** [comment]
**Furnish**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/furnish
_Blocks:_ 4 tables, 3 cabinets, chairs, stools, benches, log benches, crates, shutters, shelves
_Versions:_ 1.18.2 and 1.19.2/3
_Loaders:_ Forge

**Comment by enduum** [comment]
**Builders Crafts and Additions**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/buildersaddition
_Blocks:_ Tables, bedside tables, cupboards, small cupboards, cabinets, chairs, stools, benches, bookshelves, support brackets, kitchen counters, arcades
_Versions:_ 1.18.2 and 1.19.2/3
_Loaders:_ Forge

**Comment by AnOutdatedMind** [comment]
**Dramatic Doors***
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/buildersaddition
_Blocks_: 3 block tall doors
Versions: 1.19.2/3
Loaders: Forge

*Bit of a special case; only really apparent when installed along with Macaw's Doors and another mod that adds new types of woods (BYG in this case). Currently there are no 3 block tall doors in the style of Macaw's Doors made from BYG.

**Comment by LolJohn11** [comment]
**Shipping**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/shipping
_Blocks:_ mailboxes, wall mailboxes
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by legoatoom** [comment]
ItemSwapper
Curseforge: https://www.curseforge.com/minecraft/mc-mods/itemswapper
JSON-Files: For all wood types.
Versions: 1.19.2
Loaders: Forge/Fabric

**Comment by nitrodynamite18** [comment]
Create: Extended Flywheels
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/create-extended-flywheels
_Blocks_: flywheels, plated flywheels, and large flywheels
_Versions_: 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Corail Woodcutter
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/corail-woodcutter
_Blocks_: sawmills
_Versions_: 1.16.5, 1.17.1, 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by Dr-WeiAL** [comment]
Sophisticated Storage
Curseforge: https://www.curseforge.com/minecraft/mc-mods/sophisticated-storage
Blocks: barrel, chest
Versions: 1.18.2, 1.19.2
Loaders: Forge

**Comment by nitrodynamite18** [comment]
Create: Extended Cogs
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/create-extended-cogs
_Blocks_: cogs, large cogs
_Versions_: 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Woodworks
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/woodworks
_Blocks_: bookshelves, beehives (Also adds ladders and chests but those already exist)
_Versions_: 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Just A Raft Mod
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/just-a-raft-mod
_Entities and Items_: rafts
_Versions_: 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Hollow Woods
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/hollowwoods
_Blocks_: hollow logs, stripped hollow logs
_Versions_: 1.18.1/2, 1.19.2/3
_Loaders_: Fabric

**Comment by Aquaeyes** [comment]
Handcrafted
Curseforge: https://www.curseforge.com/minecraft/mc-mods/handcrafted
Blocks: Boards, Chairs, Tables, Benches, Couches, Beds, Dining Benches, Nightstands, Side Tables, Counters, Cupboards, Drawers, Shelves, Pillar Trims, Corner Trims
Versions: 1.19.2
Loaders: Forge/Fabric

**Comment by Numerosityy** [comment]
Verdure
Curseforge: https://www.curseforge.com/minecraft/mc-mods/verdure
Blocks: Branches
Versions: 1.18.1/2
Loaders: Forge

**Comment by Numerosityy** [comment]
Minecraft
Curseforge: N/A
Blocks: Composter, Barrel
Versions: All
Loaders: All

**Comment by Adrthegamedev** [comment]
Note that this is a quilt mod, so i'm not sure if this should happen or not:
Architecture Extensions
Modrinth: https://modrinth.com/mod/arch-ex
Blocks: Beams (Stripped Wood texture), Joists (Stripped Wood texture), Fence Posts, Crown Moldings, Post Caps, Post Lanterns
Versions: 1.19/1.19.2, 1.18.2
Loaders: Quilt

**Comment by nitrodynamite18** [comment]
Cozy
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/cozy
_Blocks_: tables, chairs, firewood
_Versions_: 1.19.1, 1.19.2
_Loaders_: Forge/Fabric

**Comment by nitrodynamite18** [comment]
Minecraft
_Curseforge_: N/A
_Blocks_: crafting table
_Versions_: All
_Loaders_: All

Biomes You'll Go has custom crafting tables for each of its woods.

**Comment by Adrthegamedev** [comment]
I Like Wood
Curseforge: https://www.curseforge.com/minecraft/mc-mods/i-like-wood
Blocks: Panels, Panels Stairs, Panels Slab, Bookshelf, Barrel, Wall, Ladder, Crafting Table, Post, Sawmill, Composter, Lectern, Chest, Torch, Soul Torch, Scaffolding, Chair, Table, Stool, Single Dresser, Log Pile, Campfire, Soul Campfire, every color of Bed, Crate
Versions: 1.19/1.19.2, 1.18.2
Loaders: Forge

This mod also adds different variations of certain items (like Item Frames or even tools) but that may be overkill to support. Note that there are also [four](https://www.curseforge.com/minecraft/mc-mods/i-like-wood-biomes-o-plenty-plugin) [different](https://www.curseforge.com/minecraft/mc-mods/i-like-wood-oh-the-biomes-youll-go-plugin) [plugin](https://www.curseforge.com/minecraft/mc-mods/i-like-wood-immersive-engineering-plugin) [mods](https://www.curseforge.com/minecraft/mc-mods/i-like-wood-twilight-forest-plugin) adding support for various other mods, and [another](https://www.curseforge.com/minecraft/mc-mods/i-like-wood-config) weird "pick and choose" form of the mod for only 1.18.2.

**Comment by curryducker** [comment]
Create: Steam 'n Rails
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/create-steam-n-rails
_Blocks_: Wooden Rails
_Versions_: 1.19.2, 1.18.2
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Bewitchment
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/bewitchment
_Items_: barks
_Versions_: 1.19.2
_Loaders_: Fabric

**Comment by nitrodynamite18** [comment]
Absent by Design
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/absent-by-design
_Blocks_: log fences, log and stripped log walls, plank walls
_Versions_: 1.19+
_Loaders_: Forge

**Comment by nitrodynamite18** [comment]
Aquaculture 2
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/aquaculture
_Blocks_: fish mounts
_Versions_: 1.18+, 1.19+
_Loaders_: Forge

**Comment by stuart0035** [comment]
**Quark**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/quark
_Blocks:_ Hollow Logs
_Versions:_ 1.19.2
_Loaders:_ Forge

**Comment by arzadu** [comment]
Quark (update)
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/quark
_Blocks_: hollow logs
_Versions_: 1.19.2+
_Loaders_: Forge

Update: completed!

**Comment by The-Bernician-Lamb** [comment]
Valhelsia Furniture
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/valhelsia-furniture
_Blocks:_ Chairs (With and without seats), Tables (With and without Table Cloths), Upholstered Chairs,  Stools (With and without seats), Desks, Desk Drawers
_Versions:_ 1.19.2+
_Loaders:_ Forge

**Comment by arzadu** [comment]
Iron Chests
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/iron-chests
_Blocks_: Chests!
_Versions_: 1.17+
_Loaders_: Forge

Adding modded chest compat would be magnificent! ex: modded wood chests (like those added by Quark) would keep their textures when upgraded to Copper, Iron, Diamond, and Netherite (Obsidian and Crystal have completely unique textures).

**Comment by nitrodynamite18** [comment]
Corail Pillar
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/corail-pillar
_Blocks_: thin pillars and full pillars
_Versions_: 1.16.5+
_Loaders_:  Forge

A bit of a weird one but still would be nice to have compatibility for.

**Comment by PengyTheGamer** [comment]
Charm
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/charm
_Blocks:_ Chests, Barrels, Bookcases, Bookshelves
_Versions:_ 1,19.2+
_Loaders:_ Fabric

Charm is a fabric mod inspired by quark, so basically similar to quark's compat

**Comment by gaforb** [comment]
Exline's Awnings
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/exlines-awnings
_Blocks_: Awnings (Plank), Awnings (Log)
_Versions_: 1.16.5, 1.17.1, 1.18.2, 1.19.2
_Loaders_: Forge, Fabric

_Notes_: A much simpler alternative to Macaw's Roofs, in my opinion. Just discovered you already support Bark Carpets by the same author as well :)

**Comment by gaforb** [comment]
Giacomo's Bookshelf
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/giacomos-bookshelf
_Blocks_: Bookshelf (Storage)
_Versions_: 1.19.2
_Loaders_: Forge

_Notes_: Can store books and, configurably, other items. Updates front texture through 4 stages (empty, slightly full, more full, completely full) depending on if 0, 1, 2, or 3+ books are stored. Drops contents when broken, unless broken with silk touch in which case it behaves like a shulker box. Given all these special behaviours it may be too difficult to implement compatibility for, though it does already have different vanilla wood types for it's shelves so that might be able to be copied/extended.

**Comment by Pipsqueak737** [comment]
**Hearth & Home**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/hearth-and-home
_Blocks:_ Parquets, trims, and sanded woods
_Versions:_ 1.19.2
_Loaders:_ Forge/Fabric

**Comment by saltyseadoggo** [comment]
**Farmer's Delight** and **all other supported mods**
_Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/farmers-delight
_Recipes:_ Cutting board recipes to turn non-vanilla signs, doors and trapdoors into their respective planks using an axe.
_Versions:_ 1.19.2
_Loaders:_ Forge, Fabric

**Comment by Synthestra** [comment]
> **Hearth & Home** _Curseforge:_ https://www.curseforge.com/minecraft/mc-mods/hearth-and-home _Blocks:_ Parquets, trims, and sanded woods _Versions:_ 1.19.2 _Loaders:_ Forge/Fabric

I'll work on this once 1.20 comes around

**Comment by LopyLuna** [comment]
**Ex Nihilo: Sequentia**
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia
_Blocks_: Sieves, Crucibles, & Barrels 
_Versions_: 1.16.5, 1.18.2, 1.19.2
_Loaders_: Forge

**Comment by ParanormalChicken** [comment]
**Chipped**
_Curseforge_: https://www.curseforge.com/minecraft/mc-mods/chipped
_Modrinth_: https://modrinth.com/mod/chipped
_Blocks_: Decorative Planks & Logs, Doors & Trapdoors, Leaves, Torches, Barrels & Crates
_Versions_: 1.18.2, 1.19.2, 1.20(.1)
_Loaders_: Forge, Fabric/Quilt

**Comment by Rocketcat19** [comment]
Backpacked
Curseforge: [https://www.curseforge.com/minecraft/mc-mods/backpacked](url)
Blocks: Backpack Shelf
Versions: 1.12.2+
Loaders: Forge

**Comment by MehVahdJukaar** [comment]
Chipped is already there

**Comment by Numerosityy** [comment]
Decoration Delight: Refurbished
Curseforge: https://www.curseforge.com/minecraft/mc-mods/decoration-delight
Blocks: Counter, Counter Corner, Counter Bend, Countertop, Stool, Mosaic
Versions: 1.18.2, 1.19.2
Loaders: Forge

**Comment by murao-kun** [comment]
Wilder Wild
Curseforge: https://www.curseforge.com/minecraft/mc-mods/wilder-wild
Blocks: Hollowed Log, Stripped Hollowed Log.
Versions: 1.19.2, 1.20.1
Loaders: Fabric

**Comment by Rocketcat19** [comment]
Lucky‘s Cozy Home
Curseforge: https://www.curseforge.com/minecraft/mc-mods/cozy-home
Blocks: Chair, Table, Bench, Stump Table, Mirror, Counter (Sink and Storage included) , Drawer, Grandfather Clock, Plank, Desk, Mirror Stand
Versions: 1.18.2, 1.19.2, 1.19.4, 1.20.1
Loaders: Forge

EDIT (XELBAYRIA): 
Won't add it because it use MCreator stuff. there are 62 classes for one type, OAK. it's not worth it.

**Comment by Xelbayria** [comment]
From now on, create a separate Issue via "New Issue", and choose "Suggest Mod", it will save you a bit of time to fill in the information: _Blocks_ and _Curseforge link_. The rest will be selected using (checkboxes) depending on the mod. 

@Rocketcat19 Don't worry about it.
